modules/ROOT/partials/misc/advcode-shortcuts.adoc

```diff
@@ -1,13 +1,21 @@
-== Advanced Code Editor search and replace keyboard shortcuts
+== Enhanced Code Editor search and replace keyboard shortcuts
 
-Advanced Code Editor provides the following shortcuts within the code dialog.
+Enhanced Code Editor provides the following shortcuts within the code dialog.
 
 [cols=",,",options="header"]
-|===
-|Action |PC |Mac
-|Find |Ctrl+F |Command+F
-|Find next instance |Ctrl+G |Command+G
-|Find previous instance |Shift+Ctrl+G |Shift+Command+G
-|Replace |Ctrl+H |Command+Option+F
-|Replace All |Shift+Ctrl+R |Shift+Command+Option+F
-|===
+|=== 
+| Action | Windows Shortcut | macOS Shortcut
+| Find | Ctrl+F | Command+F
+| Replace | Ctrl+H | Command+Option+F
+| Replace All | Shift+Ctrl+R | Shift+Command+Option+F
+| Find Next | F3 | F3
+| Find Previous | Shift+F3 | Shift+F3
+| Close Search Panel | Esc | Esc
+| Indent Code | Ctrl+] | Command+]
+| Outdent Code | Ctrl+[ | Command+[
+| Fold Code | Ctrl-Shift-[ | Command-Option-[
+| Unfold Code | Ctrl-Shift-] | Command-Option-]
+| Fold All | Ctrl-Alt-[ | Ctrl-Option-[
+| Unfold All | Ctrl-Alt-] | Ctrl-Option-]
```

Comment:
@lorenzo-pomili 
- `Ctrl-Alt-[`: Fold All is not working on windows VM 
- `Ctrl+Alt+]`: Unfold All is also not working on windows VM

They just insert `≪` and `≫` from our testing.